index.js

```diff
@@ -17,8 +18,15 @@ const newSessionHandlers = {
       this.attributes.gamesPlayed = 0;
       this.attributes.gameState = null;
     }
-    this.handler.state = states.STARTMODE;
-    this.emit(':ask', strings.welcome.output, strings.welcome.reprompt);
+    if (this.attributes.gamesPlayed === 0) {
```

Comment:
Attributes are session attributes, so gamesPlayed is a total misnomer.  gamesPlayed should be gamesPlayedInARow,  It will ALWAYS be 0 at this point in the program.  gamesPlayed is really a relic of when it was a guessing game and should be deleted.